lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
+        Parameters
+        ----------
+        positions : 1D array-like object or None.
```

Comment:
I was going into the other direction :/ (based on the fact that `LineCollection` accepts `None`). Your suggestion may be better actually:

* changing the docstring is some kind of API break but as it was raising an error anyway…
* accepting `None` is not without some side-effects on `eventplot`, which I tried to mitigate by making `eventplot(None)` equivalent to `eventplot([])`, but on second thought the API break may even be worse than the previous solution.